pymongo/synchronous/cursor.py

```diff
@@ -1285,6 +1285,16 @@ def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
         self.close()
 
     def to_list(self) -> list[_DocumentType]:
+        """Converts the contents of this cursor to a list more efficiently than ``[doc async for doc in cursor]``.
```

Comment:
```suggestion
        """Converts the contents of this cursor to a list more efficiently than ``[doc for doc in cursor]``.
```

Reply:
https://github.com/mongodb/mongo-python-driver/pull/1756 will fix the inline examples not being synchronized.